packages/portal/nuxt.config.js

```diff
@@ -39,6 +39,22 @@ const redisConfig = () => {
   return redisOptions;
 };
 
+const postgresConfig = () => {
+  const postgresOptions = {
+    enabled: featureIsEnabled('eventLogging'),
+    connectionString: process.env.POSTGRES_URL
```

Comment:
the env variable should probably be POSTGRES_CONNECTION_STRING
We could still have POSTGRES_URL/PORT/USER/PASSWORD set independently if we needed finer level control.

Reply:
It's been named `POSTGRES_URL` for consistency with our other env vars, and it is a URL, just that `pg` pkg has it named `connectionString`. There is no separate option to `pg` named `url`.

Reply:
a connectionString by any other name would establish a connection as successfully.